WordPress/ColorPalette.xcassets/Gray30.colorset/Contents.json

```diff
@@ -1,20 +1,38 @@
 {
-  "info" : {
-    "version" : 1,
-    "author" : "xcode"
-  },
   "colors" : [
     {
-      "idiom" : "universal",
       "color" : {
         "color-space" : "srgb",
         "components" : {
-          "red" : "142",
           "alpha" : "1.000",
           "blue" : "150",
-          "green" : "145"
+          "green" : "145",
+          "red" : "142"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
```

Comment:
Is that change on purpose?

Reply:
Yep. Its the dark appearance of gray30

Reply:
I'm a little bit concerned about this change.

This adds a dark color for `gray30` but I guess this is not how the Muriel system was meant to work. Your change is perfectly fine but this remembers the issues we had in #12915 (which we reverted in #12944).

What we can do is — in the code — to use:

```
UIColor(light: .gray(.shade30), dark: .gray(.shade60)).cgColor
```

This will have the same result. :)

Reply:
Excellent! Thank you for this!! <3